lettuce/simulation.py

```diff
@@ -36,12 +36,15 @@ def __init__(self, flow, lattice, collision, streaming):
     def step(self, num_steps):
         """Take num_steps stream-and-collision steps and return performance in MLUPS."""
         start = timer()
-        for _ in range(num_steps):
+        for i in range(num_steps):
```

Comment:
`i` is not used. Please change it back to `_ ` (a placeholder for an unused variable)